src/App.tsx

```diff
@@ -1,60 +1,139 @@
 import './App.scss';
+import { useEffect, useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
 export const App = () => {
+  const [titleInput, setTitleInput] = useState('');
+  const [userSelect, setUserSelect] = useState(0);
+  const [checkInput, setCheckInput] = useState(false);
+  const [isTitleError, setIsTitleError] = useState(false);
+  const [isUserError, setIsUserError] = useState(false);
+  const [visibleTodos, setVisibleTodos] = useState([...todosFromServer]);
+
+  useEffect(() => {
+    if (titleInput.length !== 0 && +userSelect !== 0) {
```

Comment:
Use `Boolean(string.length)` instead of `string.length !== 0`. Fix it everywhere

```suggestion
    if (titleInput.length !== 0 && +userSelect !== 0) {
```